src/utils.py

```diff
@@ -22,3 +24,26 @@ def valid_date(s):
     except ValueError:
         msg = "not a valid date: {0!r}".format(s)
         raise argparse.ArgumentTypeError(msg)
+
+
+class Network(Enum):
+    MAINNET = "mainnet"
+    GNOSIS = "gnosis"
+
+    def dune_v1_repr(self) -> LegacyDuneNetwork:
+        return {
+            Network.MAINNET: LegacyDuneNetwork.MAINNET,
+            Network.GNOSIS: LegacyDuneNetwork.GCHAIN,
+        }[self]
+
+    def dune_v2_repr(self) -> str:
+        return {Network.MAINNET: "ethereum", Network.GNOSIS: "gnosis"}[self]
+
+    def node_url(self) -> str:
+        return {
+            Network.MAINNET: f"https://mainnet.infura.io/v3/{os.environ['INFURA_KEY']}",
+            Network.GNOSIS: "https://rpc.gnosischain.com",
+        }[self]
+
+    def chain_id(self) -> int:
```

Comment:
Same nitpick: Missing verb. Similar suggestion as for `node_url`

Reply:
❌ - not changing - see argument above (chain id is an attribute of network) - not something which needs to be computed or even retrieved from somewhere else.

I would be willing to add a @property decorator.